File: 30 Days of Code/08 - Day 7 - Arrays.py
# ========================
#       Information
# ========================

# Direct Link: https://www.hackerrank.com/challenges/30-arrays/problem
# Difficulty: Easy
# Max Score: 30
# Language: Python

# ========================
#         Solution
# ========================

if __name__ == '__main__':
    n = int(input())

    b = list(map(int, input().split(" ")))

    for i in range(1, n+1):
        print(b[n-i], end=" ")

File: 30 Days of Code/C#/08 - Day 7 - Arrays.cs
// ========================
//       Information
// ========================

// Direct Link: https://www.hackerrank.com/challenges/30-arrays/problem
// Difficulty: Easy
// Max Score: 30
// Language: C#

// ========================
//         Solution
// ========================

using System;

class Solution {
    static void Main(String[] args) {
        Console.ReadLine();

        var str = Console.ReadLine();
        var arr = str.Split(' ');

        Array.Reverse(arr);

        foreach(var num in arr) {
            Console.Write($"{num} ");
        }
    }
}
